main.tf

```diff
@@ -348,11 +347,13 @@ module "extensions" {
   # bastion
   create_bastion_host = var.create_bastion_host
   bastion_public_ip   = local.bastion_public_ip
+  bastion_user        = var.bastion_user
   bastion_state       = var.bastion_state
 
   # operator details
   create_operator                    = var.create_operator
   operator_private_ip                = local.operator_private_ip
+  operator_user                      = var.operator_user
```

Comment:
Ditto for the operator. We need to make a change in the operator module so that an Ubuntu image can also be used.

Reply:
Added a note on setting to "opc" for supported images.